tests/e2e/flowtest/test_maintenance_pod.py

```diff
@@ -0,0 +1,178 @@
+import logging
+import time
+import random
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_ocs_version,
+    bugzilla,
+    tier2,
+    skipif_external_mode,
+)
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs.resources.deployment import (
+    get_osd_deployments,
+    get_deployments_having_label,
+    get_mon_deployments,
+)
+from ocs_ci.ocs.resources.pod import (
+    get_pods_having_label,
+    wait_for_pods_to_be_running,
+    get_operator_pods,
+)
+from ocs_ci.utility.utils import ceph_health_check
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@skipif_external_mode
+@bugzilla("2103256")
+@skipif_ocs_version("<4.12")
+class TestMaintenancePod(E2ETest):
```

Comment:
@hnallurv I wanted to know your opinion on if this test should be part of `tests/e2e/flowtest/test_maintenance_pod.py` folder?

Reply:
this should not be part of flowtest dir. let's discuss offline and decide

Reply:
this is now placed under KCS after discussing offline